packages/integration/api-report.md

```diff
@@ -567,6 +568,8 @@ export class GithubIntegration implements ScmIntegration {
   // (undocumented)
   static factory: ScmIntegrationsFactory<GithubIntegration>;
   // (undocumented)
+  isRateLimited(response: ConsumedResponse): boolean;
```

Comment:
I think it could be good to keep this a bit more open to future improvements to allow for even deeper integration with upstream rate limiting info. Thinking something like this:

```ts
interface RateLimitInfo {
  limited: boolean
}

class GithubIntegration {
  getRateLimitInfo(res: ConsumedResponse): RateLimitInfo
}
```

With that we could add more optional fields in the future that can be aligned across different providers. For example the remaining time until the limit is lifted, or remaining number of requests allowed. (although please don't add any of that now)

Reply:
Reasonable tough I would rename `limit` to `isLimited`.